docs/gds-supported-platform/tls-certificates.md

```diff
@@ -1,5 +1,12 @@
 # Public TLS Certificates
 
+All apps in GSP are protected by TLS.  By default, TLS will use a
```

Comment:
> All apps in GSP are protected by TLS.

If they're configured to...this is up to tenants...

> By default, TLS will use a cluster-provided certificate.

Again, if it's configured to do so...up to tenants to provide a `Certificate` resource

Reply:
> If they're configured to...this is up to tenants...

I mean, we only listen on port 443 and we upgrade 80 to 443.  I suppose tenants could serve plain http on port 443 but I don't think saying "this is up to tenants" would be very helpful in these docs.

> Again, if it's configured to do so...up to tenants to provide a Certificate resource

ah, I hadn't understood the nuances here. I'll update the text for this.

Reply:
> I mean, we only listen on port 443 and we upgrade 80 to 443.

Again: if the tenant namespace configuration does that; it's entirely up to them. The ingressgateway, by default, will listen on 443 and 80 (among others) but they can add other specifications via `-cluster-config`. And redirecting from 80 to 443 is controlled by the `Gateway`, which is also tenant-controlled.

Reply:
I rewrote the opening paragraph, I hope it addresses your concerns?

Reply:
👍 LGTM